src/utils.hpp

```diff
@@ -29,8 +29,12 @@
 
 char* cvk_mkdtemp(std::string& tmpl);
 int cvk_exec(const std::string& cmd, std::string* output = nullptr);
+#if ENABLE_THREAD_NAMING
 void cvk_set_thread_name_if_supported(const std::thread::native_handle_type&,
                                       const std::string&);
+#else
+#define cvk_set_thread_name_if_supported(...)
```

Comment:
Defines in headers is not very friendly, nor are lowercase defines.
Compilers might also warn about semicolons after the "call" that come without a statement.
Recommend using an empty inline function instead.

Reply:
The empty inline function will keep the `m_thread->native_handle` when the function is called in the code, which is really what's breaking google infra.

Reply:
Possible. I personally don't like the idea because it is not developer friendly.  But if @kpet likes it, I'll do that.